src/js/jwt/jwt.js

```diff
@@ -219,7 +220,7 @@ function loginAllTabs() {
 /*** User Functions ***/
 // Get user information
 exports.getUserInfo = () => {
-    log('Getting User Information');
+    log('Getting User Information changes!!!');
```

Comment:
Was the "changes!!!" string left in by mistake?

Reply:
Uhh im still working on this branch, so it was left in purposefully.